presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -654,6 +654,83 @@ This query will collect statistics for 2 partitions with keys:
 * ``partition1_value1, partition1_value2``
 * ``partition2_value1, partition2_value2``
 
+Quick Stats
+--------------------------------------
+
+The Hive connector can build basic statistics for partitions with missing statistics
+by examining file or table metadata. For example, parquet footers can be used to infer
+row counts, number of null and min/max values. These statistics help in query planning,
+and serve as interim cache of stats for partitions which haven't yet been ANALYZEd
+
+The following properties can be used to control how these quick stats are built:
+
+.. list-table::
+   :widths: 20 70 10
+   :header-rows: 1
+
+   -
+
+      - Property Name
+      - Description
+      - Default
+   -
+
+      - ``hive.quick-stats.enabled``
+      - Enable stats collection through quick stats providers. Also
+         toggleable through the ``quick_stats_enabled`` session property
```

Comment:
```suggestion
        toggleable through the ``quick_stats_enabled`` session property.
```
Every text line after the first in the Description column has one extra space indent at the beginning. I'll try to edit these here, but if you can do a local build of the docs it'd help. I'm including screenshot of the build of the current file, and a screenshot of my local edited build to show the difference. I'm sorry this is going to be a lot of comments for almost entirely single-space deleting nits! 
current
<img width="772" alt="Screenshot 2024-02-07 at 12 24 08 PM" src="https://github.com/prestodb/presto/assets/7013443/81a9ae49-212e-493e-840d-48e2406f1215">

my edited and local-built example
<img width="760" alt="Screenshot 2024-02-07 at 12 33 55 PM" src="https://github.com/prestodb/presto/assets/7013443/0d51df58-61b7-4cab-b700-78eb0eb4123d">

Reply:
My bad, I used pandoc to convert a markdown table to rst and missed this. Will fix this up